README.md

```diff
@@ -10,4 +10,4 @@ Describe how you approached to problem, and what tools and techniques you used t
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://thessans-happythoughts-api.herokuapp.com/
```

Comment:
It is very important to make your README file outstanding 😺 Most Recruiters will assess your work using it and you need to make them want to pass this repo to developers 😸